pkg/subctl/cmd/gather.go

```diff
@@ -82,23 +85,43 @@ func gatherData() {
 	err := checkGatherArguments()
 	exitOnError("Invalid arguments", err)
 
-	for module, ok := range gatherModuleFlags {
-		if ok {
-			for dataType, ok := range gatherTypeFlags {
-				if ok {
-					status := cli.NewStatus()
-					status.Start(fmt.Sprintf("Gathering %s %s...", module, dataType))
-					status.End(cli.CheckForError(gatherFuncs[module](status, dataType)))
+	configs, err := getMultipleRestConfigs(kubeConfig, kubeContext)
+	exitOnError("Error getting REST config", err)
+	for _, item := range configs {
```

Comment:
Please don't overload this patch with code for gathering from multiple clusters. That change would require its own discussion that is not within scope of this change. In this PR focus should be on gathering submariner logs, filenames, directory structures etc.

Reply:
DOne